junebug/logging_service.py

```diff
@@ -50,7 +50,7 @@ def logger_for_event(self, event):
         system = event.get('system', '-')
         parts = [self.worker_id]
         if system != '-':
-            parts.extend(system.split(','))
+            parts = system.split(',')
```

Comment:
The `system` has to have the `worker-_id` in it for us to process the log, so if we don't drop it, we end up with `worker_id.something_with_worker_id_in_it`, most of the time it will be `worker_id.worker_id`, which seemed unnecessary.